src/_pytest/assertion/util.py

```diff
@@ -368,15 +377,17 @@ def _compare_eq_set(
 
 
 def _compare_eq_dict(
-    left: Mapping[Any, Any], right: Mapping[Any, Any], verbose: int = 0
+    left: Mapping[Any, Any], right: Mapping[Any, Any], verbose: int = 0,
 ) -> List[str]:
     explanation = []  # type: List[str]
     set_left = set(left)
     set_right = set(right)
     common = set_left.intersection(set_right)
     same = {k: left[k] for k in common if left[k] == right[k]}
     if same and verbose < 2:
-        explanation += ["Omitting %s identical items, use -vv to show" % len(same)]
+        explanation += [
```

Comment:
Changing `%` to `format` is an unrelated change as well, so would change it back to the way it was just to keep the diff focused.